web/viewer.css

```diff
@@ -539,164 +740,121 @@ html[dir='rtl'] .splitToolbarButton > .toolbarButton {
+  padding: 10px 0;
   width: 1px;
-  background-color: rgba(0, 0, 0, 0.5);
+  background-color: var(--separator-color);
   z-index: 99;
-  box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.08);
   display: inline-block;
-  margin: 5px 0;
+  margin: 4px 0;
 }
+
+.findbar .splitToolbarButtonSeparator {
+  background-color: var(--field-border-color);
+  margin: 0;
+  padding: 13px 0;
+}
+
 html[dir='ltr'] .splitToolbarButtonSeparator {
   float: left;
 }
 html[dir='rtl'] .splitToolbarButtonSeparator {
   float: right;
 }
-.splitToolbarButton:hover > .splitToolbarButtonSeparator,
-.splitToolbarButton.toggled > .splitToolbarButtonSeparator {
-  padding: 12px 0;
-  margin: 1px 0;
-  box-shadow: 0 0 0 1px rgba(255, 255, 255, 0.03);
-}
 
 .toolbarButton,
 .dropdownToolbarButton,
 .secondaryToolbarButton,
 .overlayButton {
   min-width: 16px;
   padding: 2px 6px 0;
-  border: 1px solid rgba(0, 0, 0, 0);
+  border: none;
   border-radius: 2px;
-  color: rgba(255, 255, 255, 0.8);
+  color: var(--main-color);
   font-size: 12px;
   line-height: 14px;
   user-select: none;
   /* Opera does not support user-select, use <... unselectable="on"> instead */
   cursor: default;
+  box-sizing: border-box;
 }
 
 html[dir='ltr'] .toolbarButton,
 html[dir='ltr'] .overlayButton,
 html[dir='ltr'] .dropdownToolbarButton {
-  margin: 3px 2px 4px 0;
+  margin: 2px 1px;
 }
 html[dir='rtl'] .toolbarButton,
 html[dir='rtl'] .overlayButton,
 html[dir='rtl'] .dropdownToolbarButton {
-  margin: 3px 0 4px 2px;
+  margin: 2px 1px;
 }
 
+html[dir='ltr'] #toolbarViewerLeft > .toolbarButton:first-child,
+html[dir='rtl'] #toolbarViewerRight > .toolbarButton:last-child {
+  margin-left: 2px;
+}
+
+html[dir='ltr'] #toolbarViewerRight > .toolbarButton:last-child,
+html[dir='rtl'] #toolbarViewerLeft > .toolbarButton:first-child {
+  margin-right: 2px;
+}
 .toolbarButton:hover,
 .toolbarButton:focus,
-.dropdownToolbarButton,
 .overlayButton,
 .secondaryToolbarButton:hover,
 .secondaryToolbarButton:focus {
-  background-color: rgba(0, 0, 0, 0.12);
-  background-image: linear-gradient(rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
-  background-clip: padding-box;
-  border: 1px solid rgba(0, 0, 0, 0.35);
-  border-color: rgba(0, 0, 0, 0.32) rgba(0, 0, 0, 0.38) rgba(0, 0, 0, 0.42);
-  box-shadow: 0 1px 0 rgba(255, 255, 255, 0.05) inset,
-              0 0 1px rgba(255, 255, 255, 0.15) inset,
-              0 1px 0 rgba(255, 255, 255, 0.05);
-}
-
-.toolbarButton:hover:active,
-.overlayButton:hover:active,
-.dropdownToolbarButton:hover:active,
-.secondaryToolbarButton:hover:active {
-  background-color: rgba(0, 0, 0, 0.2);
-  background-image: linear-gradient(rgba(255, 255, 255, 0.05), rgba(255, 255, 255, 0));
-  border-color: rgba(0, 0, 0, 0.35) rgba(0, 0, 0, 0.4) rgba(0, 0, 0, 0.45);
-  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1) inset,
-              0 0 1px rgba(0, 0, 0, 0.2) inset,
-              0 1px 0 rgba(255, 255, 255, 0.05);
+  background-color: var(--doorhanger-hover-color);
```

Comment:
Did you actually mean `--button-hover-color` here!?

Given that e.g. `.splitToolbarButton > .toolbarButton:hover` uses that css variable, the effect of *this* line is that different toolbar buttons end up looking slightly different when hovered.
Is this really intended, since it feels a bit inconsistent?